client/schema_field_description.go

```diff
@@ -154,7 +154,11 @@ func (k ScalarArrayKind) Underlying() string {
 }
 
 func (k ScalarArrayKind) IsNillable() bool {
```

Comment:
I'm actually not sure how the tests did not fail before. I'm still investigating, but I believe its because of encoding / decoding of `[]int{}` vs `nil`.

Reply:
I've added a unit test, but I don't think an integration test can actually catch this problem due to the encoding I mentioned above.

Reply:
https://github.com/sourcenetwork/defradb/issues/2539